rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -428,7 +428,10 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
                             isNull('dateCompleted')
                             if(!query.considerPostponedRunsAsRunningFilter){
                                 le('dateStarted', now)
-                                ne('status', EXECUTION_SCHEDULED)
+                                or {
+                                    ne('status', EXECUTION_SCHEDULED)
```

Comment:
should this be changed to `eq`?
```suggestion
                                    eq('status', EXECUTION_SCHEDULED)
```

The original query (status!='scheduled') meant to show only currently running jobs, but missed the case where a scheduled execution had started.  
the new query should include (status==null) which is normal executions, as well as (status=='scheduled') which is scheduled executions that have started.

Reply:
@gschueler When jobs is scheduled with `Run job later` option, the status is not defined as 'scheduled'. Previously, the query was only checking if the date started was before then 'now' and status different of 'scheduled' to avoid that `Run job later` was considered as running job. Now I just include the `isNull` clause on query.

Reply:
yes, status is set to `scheduled` when you use run job later.

later if the execution starts, it seems it is still status=scheduled.  so you can have a running execution with status 'scheduled'